pkg/bus/monitor.go

```diff
@@ -11,61 +10,10 @@ import (
 	"go.uber.org/fx"
 )
 
-const (
-	CommittedTransactions string = "COMMITTED_TRANSACTIONS"
-	SavedMetadata         string = "SAVED_METADATA"
-	UpdatedMapping        string = "UPDATED_MAPPING"
-	RevertedTransaction   string = "REVERTED_TRANSACTION"
-)
-
 type ledgerMonitor struct {
 	publisher *sharedpublish.TopicMapperPublisher
 }
 
-func (l *ledgerMonitor) publish(ctx context.Context, ledger string, et string, data interface{}) {
```

Comment:
Why remove this method? 
There are a useless copy/paste on all others methods now.

Reply:
still experimenting!